ext/drreg/drreg.c

```diff
@@ -1985,13 +1988,30 @@ is_aflags_spill(instr_t *inst)
 #endif
 }
 
+static reg_id_t
+aflags_spill_get_reg(instr_t *inst)
```

Comment:
Document that this assumes the given instr is one that spills aflags.

Reply:
Done. And `aflags_restore_get_reg` likewise!